_templates/sitewide.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 require_once __DIR__.'/../backend/lib/autoload.php';
+require_once __DIR__.'/../backend/log-echo.php';
 require_once __DIR__.'/../backend/config.loader.php';
 
 // Setup sentry error logging
```

Comment:
considering sentry is already setup with the `log-echo` file, we could remove the next couple of lines